src/process_utility.c

```diff
@@ -1712,16 +1722,32 @@ process_utility_AtEOXact_abort(XactEvent event, void *arg)
 	}
 }
 
+static void
+process_utility_AtSubXact_abort(SubXactEvent event, SubTransactionId mySubid,
+								SubTransactionId parentSubid, void *arg)
+{
+	switch (event)
+	{
+		case SUBXACT_EVENT_ABORT_SUB:
+			expect_chunk_modification = false;
```

Comment:
I think I understand why resetting this bool is needed, but should probably clarify in a comment.

Reply:
fixed